client/state/jetpack-connect/constants.js

```diff
@@ -1,16 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import { shouldShowOfferResetFlow } from 'lib/abtest/getters';
+import { JETPACK_RESET_PLANS } from 'lib/plans/constants';
+import { JETPACK_PRODUCTS_LIST } from 'lib/products-values/constants';
+import { UPSELL_PRODUCT_MATRIX } from 'my-sites/plans-v2/constants';
 export const JETPACK_CONNECT_TTL = 60 * 60 * 1000; // 1 hour
 export const JETPACK_CONNECT_TTL_SECONDS = JETPACK_CONNECT_TTL / 60;
 export const JETPACK_CONNECT_AUTHORIZE_TTL = 60 * 60 * 1000; // 1 hour
 export const AUTH_ATTEMPS_TTL = 60 * 1000; // 1 minute
 
+// Offer Reset allows users to purchase two products at the same time. The allowed
+// combinations are stored in `OFFER_RESET_COMBINED_FLOW_TYPES`. From this matrix,
+// we get a list of strings that follow the `productSlug1,productSlug2` pattern.
+export const OFFER_RESET_COMBINED_FLOW_TYPES = shouldShowOfferResetFlow()
```

Comment:
We need to allow combinations such as `jetpack_scan,jetpack_backup_daily` because they are part of the upsell flows.